lib/inventory_sync/async/inventory_hosts_sync.rb

```diff
@@ -4,6 +4,13 @@ class InventoryHostsSync < QueryInventoryJob
       set_callback :iteration, :around, :setup_facet_transaction
       set_callback :step, :around, :create_facets
 
+      def plan
+        concurrence do
```

Comment:
Concurrence is the default, there's no need to be explicit about it

Reply:
Removed redundant call